File: Problem2/problem2.py
a, b = 1, 2
next_no, s = 0, 0

while a < 4000000:
	if a % 2 == 0:
		s = s + a
	next_no = a+b
	a = b
	b = next_no


print("Sum = ", s) 

File: Problem2/problem2.c
#include <stdio.h>

int main()
{
	unsigned int a, b, next, sum;

	a = 1;
	b = 2;
	next = 0;
	sum = 0;

	while (a < 4000000) {
		if (a % 2 == 0)
			sum = sum + a;

		next = a + b;
		a = b;
		b = next;
	}

	printf("Sum = %d\n", sum);
}
